### PR TITLE
Feature/python binding 

### PR DESCRIPTION
**Pull request by haixuanTao**
- Add another crate for python binding
- Create a Github workflow to push precompilation to pypi
- Layout groundwork for backend conformity test although most tests are not passing at the moment.

### CODE REVIEW

File: py-wonnx/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+use std::collections::HashMap;
+
+use wonnx::Session;
+#[pyclass]
+#[repr(transparent)]
+pub struct PySession {
```
Comment (pixelspark) [inline py-wonnx/src/lib.rs:8]: Could make this class name simply 'Session' in Python by using `#[pyclass(name="Session")]` (see [here](https://pyo3.rs/master/class.html#customizing-the-class)) :-)

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
```
Comment (pixelspark) [inline Cargo.toml:1]: After this PR the main `wonnx` package is in the root whereas the python stuff is in a subdirectory. A more 'rusty' way of organising the repository is using a [workspace](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html). This basically means you have a directory for 'wonnx' (with its Cargo.toml), a directory for 'wonnx-python' (with its Cargo.toml) and a Cargo.toml in the root specifying 'wonnx' and 'wonnx-python' as workspace members.

File: py-wonnx/src/lib.rs (hunk 1 above)
Comment (haixuanTao) [reply py-wonnx/src/lib.rs:8]: Yeah, the thing is I may wrap PySession within a Python Class called Session, to add other functionality, sort of like in polars: https://github.com/pola-rs/polars/blob/master/py-polars/polars/internals/frame.py

Thanks for the suggestion though. Next time I'll run the PR by you.

File: Cargo.toml (hunk 2 above)
Comment (haixuanTao) [reply Cargo.toml:1]: Yes, you're right. I'll fix that later.

**Pull request merged**